nautobot/dcim/api/serializers.py

```diff
@@ -199,6 +199,19 @@ class Meta:
         list_display_fields = ["name", "status", "parent", "tenant", "description", "tags"]
         # https://www.django-rest-framework.org/api-guide/validators/#optional-fields
         validators = []
+        # TODO: Add documentation of this schema
+        detail_view_config = [
+            {
+                "Location": {"fields": ["name", "rack_count", "parent"]},
+                "Management": {"fields": ["status", "prefix_count", "device_count"]},
```

Comment:
why is `rack_count` in `Location` but `prefix_count` and `device_count` in Management? Any specific reasoning?

Reply:
Nope, Could not think of a better grouping,
Illd use the grouping on the legacy UI 👍